kubejs/server_scripts/enigmatica/kubejs/base/recipes/shaped.js

```diff
@@ -376,6 +376,9 @@ events.listen('recipes', function (event) {
             A: '#forge:wax',
             B: '#forge:string'
         }),
+        shapedRecipe(item.of('byg:embur_hyphae', 3), ['AA ', 'AA '], {

```

Comment:
This pattern will result in the recipe only working on the left side of the crafting grid `['AA', 'AA']` would be better imo :)

Reply:
Right! Will change that